test/e2e/framework/pv_util.go

```diff
@@ -78,14 +78,29 @@ type PVCMap map[types.NamespacedName]pvcval
 //	 	},
 //	 }
 type PersistentVolumeConfig struct {
-	PVSource         v1.PersistentVolumeSource
-	Prebind          *v1.PersistentVolumeClaim
+	// [Optional] NamePrefix defaults to "pv-" if unset
+	NamePrefix string
+	// [Optional] Labels contains information used to organize and categorize
+	// objects
+	Labels labels.Set
+	// PVSource contains the details of the underlying volume and must be set
+	PVSource v1.PersistentVolumeSource
```

Comment:
Just checking, this is the only required field?

Reply:
yep, everything else is optional